lib/libsys/statfs.2

```diff
@@ -106,8 +106,8 @@ All I/O to the file system is done synchronously.
 .It Dv MNT_ASYNC
 No file system I/O is done synchronously.
 .It Dv MNT_SOFTDEP
-Soft updates being done (see
-.Xr ffs 7 ) .
+Soft updates being done
+.Pq see Xr ffs 4 .
```

Comment:
Please leave the markup the way it was. If you absolutely want to change it (and the approximately 600 other occurrences) you can do it in a separate PR and use `Po` / `Pc`.

Reply:
I'm not kidding about 600 btw:

````
% git grep '\(see$' '*.[0-9]' | wc -l
     590
````